src/rules/lowercase-name.ts

```diff
@@ -118,13 +141,20 @@ export default createRule<
       },
     ],
   } as const,
+
   defaultOptions: [{ ignore: [], allowedPrefixes: [] }],
   create(context, [{ ignore = [], allowedPrefixes = [] }]) {
     return {
       CallExpression(node) {
         if (!isJestFunctionWithLiteralArg(node)) {
           return;
         }
+
+        const topMethod = jestTopFunctioName(node, isTopMostJestFunction(node));
+        if (topMethod && ignore.includes(TopCase.top)) {
```

Comment:
Could you replace `TopCase.top` w/ an `ignoreTopLevel` option, [as](https://github.com/jest-community/eslint-plugin-jest/pull/428/files#r331757372) previously [discussed](https://github.com/jest-community/eslint-plugin-jest/pull/428/files#r332680009)?

You can also inline `topMethod`, since it's not used elsewhere :)